service/ziti-tunnel/service/state.go

```diff
@@ -497,6 +503,17 @@ func (t *RuntimeState) RemoveRoute(destination net.IPNet, nextHop net.IP) error
 }
 
 func (t *RuntimeState) Close() {
+	val := tun_release.Load()
+	if val != nil {
```

Comment:
you should just initialize it and make sure it can't be nil so you never need the nil check. you also don't need 'if' statements to print the state, just print that state in one statement before these ifs. you should also just check to see if it's in the process of closing or already closed here. that brings up another point, we really only care if this is in the process of being closed or is already closed so that if close it called twice it just skips out. i don't think 'running' or 'success' or 'failed' are really relevant? are those states relevant?